projects/packages/jetpack-mu-wpcom/src/features/wpcom-widgets/class-jetpack-i-voted-widget.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
Could this be a git mv rather than an addition and deletion?

Reply:
What's weird is that [my first commit _was_ a `git mv`](https://github.com/Automattic/jetpack/pull/42924/commits/d2aed4e6b3754547e8e3b505456b4d06df1a1ac3), and then it later changed to considering it deleted...

Reply:
Maybe it's just a github reporting thing? I went through the commits 1 by 1 and none of them deleted the file. The only one that references it is that first commit which says it was a move.